js/src/components/SEMrushRelatedKeyphrasesModal.js

```diff
@@ -5,7 +5,8 @@ import { __ } from "@wordpress/i18n";
 import PropTypes from "prop-types";
 
 /* Yoast dependencies */
-import { BaseButton } from "@yoast/components";
+import { Button } from "@yoast/components/src/button";
+import { ButtonStyledLink } from "@yoast/components";
```

Comment:
Wondering whether these two imports can be merged in just one import.

Reply:
Yes, the JS repo has some changes ongoing but I think we can merge into one:
```
import { Button, ButtonStyledLink } from "@yoast/components/src/button";
```
Will do.